storage/rocksdb/rdb_native_dd.h

```diff
@@ -21,13 +21,17 @@
 /* MySQL header files */
 #include "sql/dd/object_id.h"
 
+typedef unsigned int uint;
```

Comment:
I am sorry, I didn't realize you cannot forward declare a simple `typedef`. While repeating a `typedef` is correct, it's probably better to include the header in this case.